libraries/AP_DAC/AP_DAC_MCP40D1x.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+  DAC backend for MCP40D1x I2C DACs
+ */
+#include "AP_DAC_config.h"
+
+#if AP_DAC_ENABLED
```

Comment:
```suggestion
#if AP_DAC_MCP40D1X_ENABLED
```

... and similarly for the other backend; this should be defined in `AP_DAC_config.h`

Reply:
Fixed